### PR TITLE
Add Net schema and resource support

### PR DESCRIPTION
**Pull request by sjberman**
First pass of BIG-IP net resource support for CCCL. This includes a new schema for net resources,
new Net resource classes (Arp and FDBTunnel), and associated unit tests. LTM and NET will be
processed separately, so some ltm-specific operations have been renamed. The API will also change
to have an "apply_ltm_config" and "apply_net_config".

### CODE REVIEW

**Comment by sjberman** [comment]
I haven't yet changed the external API to use "apply_ltm_config" instead of "apply_config", but that can be added to these changes if we see fit.

File: f5_cccl/schemas/tests/net_service.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (edarzins) [inline f5_cccl/schemas/tests/net_service.json]: Stick with spaces instead of tabs in the project.

File: f5_cccl/test/bigip_net_data.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
```
Comment (edarzins) [inline f5_cccl/test/bigip_net_data.json]: Stay with spaces instead of tabs in this project.

File: f5_cccl/bigip.py [hunk 3]
```diff
@@ -70,25 +75,7 @@ def __init__(self, bigip, partition, prefix=None, manage_types=None):
         if prefix:
             self._prefix = prefix
 
-        # This currently hard codes what resources we care about until we
```
Comment (edarzins) [inline f5_cccl/bigip.py]: Thanks for getting rid of dead code!

File: f5_cccl/api.py [hunk 4]
```diff
@@ -58,20 +59,20 @@ def __init__(self, bigip, partition, prefix=None, schema_path=None):
...
+        """Apply LTM service configurations to the BIG-IP partition.
 
         :param services: A serializable object that defines one or more
-        services. Its schema is defined by cccl-api-schema.json.
+        services. Its schema is defined by cccl-ltm-api-schema.json.
 
         :return: True if successful, otherwise an exception is thrown.
         """
-        return self._service_manager.apply_config(services)
+        return self._service_manager.apply_ltm_config(services)
```
Comment (edarzins) [inline f5_cccl/api.py]: Add interface for apply_net_config() too!

File: f5_cccl/service/test/test_validation.py [hunk 5]
```diff
@@ -43,13 +43,13 @@ class TestConfigValidator(object):
 
     @pytest.fixture(autouse=True)
     def default_schema(self):
-        with open('f5_cccl/schemas/cccl-api-schema.yml', 'r') as fp:
+        with open('f5_cccl/schemas/cccl-ltm-api-schema.yml', 'r') as fp:
```
Comment (edarzins) [inline f5_cccl/service/test/test_validation.py]: Validate net schema too.

File: f5_cccl/api.py (hunk 4 above)
Comment (sjberman) [reply f5_cccl/api.py]: Done

File: f5_cccl/schemas/tests/net_service.json (hunk 1 above)
Comment (sjberman) [reply f5_cccl/schemas/tests/net_service.json]: Done

File: f5_cccl/test/bigip_net_data.json (hunk 2 above)
Comment (sjberman) [reply f5_cccl/test/bigip_net_data.json]: Done

**Pull request merged**